src/toil/jobStores/abstractJobStore.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, config=None):
             config.workflowID = str(uuid4())
             self.__config = config
             self.writeConfigToStore()
+        logger.info('The workflow ID is: %s' %self.__config.workflowID)
```

Comment:
Unless you feel the only reason one would want to know the workflow dir is in a debugging situation